docs/data/xrootd/overview.md

```diff
@@ -27,3 +27,11 @@ An [XRootD standalone server](/data/xrootd/install-standalone) exports an existi
 using both the XRootD and WebDAV protocols.
 Generally, only sites affiliated with large VOs would need to install an XRootD standalone server so consult your VO if
 you are interested in contributing storage.
+
+XRootD Storage Element
+----------------------
+
+An [XRootD storage element (SE)](/data/xrootd/install-storage-element) is a network storage solution that enables
```

Comment:
Consider rephrasing this paragraph (and the copy of it a few lines above) to put the 'if statement' first -- something like "If your site is affiliated with a large VO or you wish to contribute storage, you might need to install an XRootD storage element (SE), which is a network storage solution..."

Actually, I have no idea what you mean by "network storage solution" or how this differs from a standalone server. You always need an "existing file system" whether that's HDFS or ext4. (I'd mention this in the previous section but it's not part of the PR so GitHub doesn't let me comment on it.)

Reply:
I generally agree with putting the "if" part first but it looks a little goofy doing it before telling people what the actual product is. Also you should really only be installing XRootD standalone/SE if the VO you support wants you to.

I've updated the phrasing of the standalone storage solution. Basically the difference is:

1. Have HDFS/Ceph/Lustre/etc already? Use XRootD standalone.
1. Don't have HDFS/Ceph/Lustre/etc already? Use XRootD SE.

I think it's clear how it's written but I'm open to suggestions.

Reply:
Item 2 is not spelled out anywhere. How about:
"An XRootD Storage Element (SE) exports a local filesystem such that a VO can access it using the XRootD and WebDAV protocols."

Reply:
Alternatively, instead of having separate sections (some of which are rather short), have a list:

The OSG supports multiple configurations of XRootD:
1. XCache cache, which can speed up data transfer for jobs that have cacheable data - see overview. ATLAS and CMS have their own implementations.
2. XCache origin, which can serve data to XCache caches. Set one up if your users have data that may benefit from being cached - see overview. ATLAS and CMS have their own implementations.
3. XRootD standalone, which can serve data via WebDAV or the XRootD protocol from an existing distributed file system such as HDFS or Lustre. Set one up if you want to provide storage for a large VO, and already have a distributed file system; contact your VO first.
4. XRootD Storage Element (SE), which can serve data via WebDAV or the XRootD protocol from a local file system. Set one up if you want to provide storage for a large VO, and do not have a distributed file system; contact your VO first.

Reply:
> "An XRootD Storage Element (SE) exports a local filesystem such that a VO can access it using the XRootD and WebDAV protocols."

This seems to imply a solution using XRootD to serve data from a single host. A standalone setup would be more appropriate (i.e. an XRootD door that provides access to an existing FS) since the SE setup runs a `cmsd`, which benefits distributed installations. AFAICT, no one actually runs a storage solution using a single host so I'm not sure it's worth mentioning.

> Alternatively, instead of having separate sections (some of which are rather short), have a list:

In principle I agree but separating them into sections makes it easier for users to differentiate between them. I'm also on the fence on separating out the XCache cache/origin because it really only applies to the StashCache implementation and so that's a detail that's more appropriate for the StashCache overview.